files/preStudies/frameworks.tex

```diff
@@ -0,0 +1,80 @@
+\section{Frameworks}
+
+This section will evaluate different frameworks for cross-platform development which could be used in this project. Some \textit{must have} and \textit{nice to have} capabilities will be listed below.
+	
+\textbf{Must have:}
+	\begin{itemize}
+	\item Cross platform capabilities.
+	\item Wide community support.
+	\item \textit{Bluetooth Low Energy} capabilities.
+	\end{itemize}
+	
+\textbf{Nice to have:}
+	
+	\begin{itemize}
+	\item 1 platform both for web and apps.
+	\item Type safety.
+	\item Reactive data. Updated data in the database should reflect on the client.
+	\item Compiles to native code for increased performance.
+	\end{itemize}
+	
+
+\subsubsection{Meteor}
+
+\textbf{Language:} JavaScript
+\\
+\textbf{Description:}
+\\
+\textit{Meteor} is a platform and framework for developing both web applications as well as mobile applications. It comes with \textit{Cordova} built-in and is considered super fast for developing applications and prototyping. It comes with a lot of build-in tools and a package library for easy setup of things like account/user integration and login integrations of different kinds like Google and Facebook. It also comes with hot code push which is a tool for uploading new versions of your apps to the stores. \textit{Meteor} also have its own Command Line Interface which is very useful when developing. \textit{Meteor} also allowes you to write your backend in plain javaScript. Out of the box it integrates with Mongo, a document database for data storage. It also uses DDP (Distributed Data Protocol) for sending data between the client and the server. This data is reactive and will be updated at the clients when it changes.
+\\
+\textit{Meteor} comes with its own view layer named Blaze. This can be substituted with React or AngularJS if either of them are preferred.
+
+\subsubsection{NativeScript}
+\textbf{Language:} JavaScript
+\\
+\textbf{Description:}
+\\
+\textit{NativeScript} uses a javascript toolchain to compile the javascript code into native API’s. This is a huge advantage because it makes the app work exactly like it was written natively without ever writing any native code or knowing about the native API’s. The markup is written in XML which is then styled with CSS.
+It’s developed by Telerik, a big software company where you can opt-in for support through a paid license. It’s supported by Google and therefore works well with \textit{Angular} and TypeScript. With TypeScript, the ''nice to have'' feature, type safety is fulfilled. \textit{Bluetooth Low Energy} capabilities can be achieved with the npm package \textit{nativescript-bluetooth}, created by npm user eddyverbruggen. \textit{NativeScript} supports iOS, Android, \textit{Universal Windows Platform} and if \textit{Angular} is used it also supports building web applications \citep{preStudy:frameworks:nativescript}.
```

Comment:
It's -> It is
''nice to have'' in italics instead of quotes
no comma after feaure
npm package should be Node.js package as NPM is just the manager CLI.
iOS and Android in italics